WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -393,6 +396,18 @@ public static int getThemeImageSizeWidth() {
         }
     }
 
+    // Wizard-style login flow
```

Comment:
I don't understand why you need to store an activated state here, can we only use the feature flag instead? Or are you planning to change the sign in behavior remotely?